internal/pkg/redis/goredis.go

```diff
@@ -128,7 +129,12 @@ func (g *goRedisWrapper) getSubscription(topic string) *goRedis.PubSub {
 	defer g.subscriptionsMutex.Unlock()
 	subscription, exists := g.subscriptions[topic]
 	if !exists {
-		subscription = g.wrappedClient.PSubscribe(topic)
+		// subscribe two topics for empty and many suffix levels to match MQTT multi-level wildcard
```

Comment:
Please add `why` comment to explain why this is needed, not what the code is doing which is understood by reading the code.